docs/user-manuals/multi-hierarchy-elastic-quota-management.md

```diff
@@ -616,6 +618,12 @@ rules:
   - patch
   - update
   - delete
+- apiGroups:
+  - ""
+  resources:
+  - pods/eviction
```

Comment:
Actually I'm not sure if this should be installed by default. Why do we need users to configure this manually:)

Reply:
We will add the RBAC in the latest version.